src/qml/models/sendrecipient.cpp

```diff
@@ -70,6 +70,9 @@ bool SendRecipient::subtractFeeFromAmount() const
 CAmount SendRecipient::cAmount() const
 {
     // TODO: Figure out who owns the parsing of SendRecipient::amount to CAmount
+    if (m_amount == "") {
+        return 0;
+    }
```

Comment:
Why is this necessary?

The Qt docs [states](https://doc.qt.io/qt-6/qstring.html#toLongLong) that `QString::toLongLong()`:
> Returns 0 if the conversion fails.